components/org.wso2.is.configuration.migrater/src/main/resources/bin/config-migrate.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+# ----------------------------------------------------------------------------
+#  Copyright 2019 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+
+#=================The start of the script:============================================
+BASE_DIR="$(cd "$(dirname "$0")"; pwd)";
+
+echo " =================================================================================== "
+echo " |   ----------------------------------------------                                | "
+echo " |   Welcome to the new config model migration tool                                | "
+echo " |  ----------------------------------------------                                 | "
+echo " |                                                                                 | "
+echo " =================================================================================== "
+echo " "
+while :
+do
+        echo "Please enter older IS home path that need to be migrated :"
```

Comment:
Shall we change this to "Please enter the older IS home path that needs to be migrated :"